test/functional/test_nvme_hotplug.py

```diff
@@ -0,0 +1,281 @@
+                },
+                {
+                    "addr": "8.0",
+                    "bus": "pcie.0",
+                    "chassis": 1,
+                    "device": "ioh3420",
+                    "id": "root_port2",
+                    "pri_bus": 0,
+                    "sec_bus": 60,
+                    "slot": 3
+                }
+            ],
+            "switch": [
+                {
+                    "downstream": [
+                        {
+                            "addr": "2.0",
+                            "bus": "upstream1",
+                            "chassis": 1,
+                            "device": "xio3130-downstream",
+                            "id": "downstream1",
+                            "slot": 190,
+                            "pri_bus": 41,
+                            "sec_bus": 42
+                        },
+                        {
+                            "addr": "3.0",
+                            "bus": "upstream1",
+                            "chassis": 1,
+                            "device": "xio3130-downstream",
+                            "id": "downstream2",
+                            "slot": 160,
+                            "pri_bus": 41,
+                            "sec_bus": 43
+                        }
+                    ],
+                    "upstream": [
+                        {
+                            "bus": "root_port1",
+                            "device": "x3130-upstream",
+                            "id": "upstream1"
+                        }
+                    ]
+                },
+                {
+                    "downstream": [
+                        {
+                            "addr": "2.0",
+                            "bus": "upstream2",
+                            "chassis": 1,
+                            "device": "xio3130-downstream",
+                            "id": "downstream3",
+                            "slot": 193,
+                            "pri_bus": 61,
+                            "sec_bus": 62
+                        },
+                        {
+                            "addr": "3.0",
+                            "bus": "upstream2",
+                            "chassis": 1,
+                            "device": "xio3130-downstream",
+                            "id": "downstream4",
+                            "slot": 164,
+                            "pri_bus": 61,
+                            "sec_bus": 63
+                        }
+                    ],
+                    "upstream": [
+                        {
+                            "bus": "root_port2",
+                            "device": "x3130-upstream",
+                            "id": "upstream2"
+                        }
+                    ]
+                }
+            ]
+        }
+        node = model.CNode(conf)
+        node.init()
+        node.precheck()
+        node.start()
+        time.sleep(3)
```

Comment:
Try to avoid 'sleep', there is a func: wait_node_up() in infrasim-compute/infrasim/model/core/node.py, can we replace with wait_node_up()?

Reply:
Hi Echo,I fixed it. Please review it. Thanks.